CONTRIBUTING.md

```diff
@@ -71,8 +85,16 @@ Skip any steps you've already completed on your own. This is meant as an exhaust
 1. Follow the environment set up instructions in the README or CONTRIBUTING document in the repository
 1. Start working on the issue you chose 🎉
 
-## 🏃 Good first issues
+### 🏃 Good first issues
 
 Most of these issues are potentially able to be completed in less than 4 hours, development environment set up included. It may take significantly more or less than 4 hours depending on experience and how smoothly development environment set up goes. Unfortunately sometimes dev env set up can sometimes be tricky! In these cases it would be helpful to the Openverse project to share your experience in an issue so we can try to remove any roadblocks for future contributors. **You can make meaningful contributions to the project in the form of valuable feedback about the development experience!**
 
 [List of Good First Issues](https://github.com/search?q=label%3A%22good+first+issue%22+state%3Aopen+is%3Aissue+repo%3Awordpress%2Fopenverse+repo%3Awordpress%2Fopenverse-frontend+repo%3Awordpress%2Fopenverse-api+repo%3Awordpress%2Fopenverse-catalog+label%3A%22good+first+issue%22+state%3Aopen+repo%3Awordpress%2Fopenverse&type=Issues)
+
+## 📖 Translation contributions
+
+You can also contribute to Openverse by translating it.
+
+An overview of Openverse translations is here: [https://translate.wordpress.org/projects/meta/openverse/](https://translate.wordpress.org/projects/meta/openverse/)
+
+A getting started guide for translating on GlotPress (the software behind [translate.wordpress.org](http://translate.wordpress.org)) is here: [https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org/#getting-started](https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org/#getting-started)
```

Comment:
Considering how long the "code contribution" section is, it might be nice to move this to the middle of the list after design _or_ add a Table of Contents to the top of the document so that people scanning will notice it easy.